### PR TITLE
Build: Unify custom dockerfiles, fix missing semicolon

### PR DESCRIPTION
**Pull request by DanCech**
While fixing the typo @grongor pointed out in #58101 I realized that we don't really need 2 custom dockerfiles either, this PR unifies them into a single file that can be used to build alpine or ubuntu-based custom images.

### CODE REVIEW

File: docs/sources/setup-grafana/installation/docker/index.md [hunk 1]
```diff
@@ -113,7 +113,20 @@ docker run -d \
 
 You can build your own customized image that includes plugins. This saves time if you are creating multiple images and you want them all to have the same plugins installed on build.
 
-In the [Grafana GitHub repository](https://github.com/grafana/grafana) there is a folder called `packaging/docker/custom/`, which includes two Dockerfiles, `Dockerfile` and `ubuntu.Dockerfile`, that can be used to build a custom Grafana image. It accepts `GRAFANA_VERSION`, `GF_INSTALL_PLUGINS`, and `GF_INSTALL_IMAGE_RENDERER_PLUGIN` as build arguments.
+In the [Grafana GitHub repository](https://github.com/grafana/grafana) there is a folder called `packaging/docker/custom/`, which includes a Dockerfile that can be used to build a custom Grafana image. It accepts `GRAFANA_VERSION`, `GF_INSTALL_PLUGINS`, and `GF_INSTALL_IMAGE_RENDERER_PLUGIN` as build arguments.
+
+The `GRAFANA_VERSION` build argument needs to be a valid `grafana/grafana` docker image tag. By default this will build an Alpine-based image. To build an Ubuntu-based image, append `-ubuntu` to the `GRAFANA_VERSION` build argument (Grafana v6.5+).
+
+Example of how to build and run:
```
Comment (chri2547) [inline docs/sources/setup-grafana/installation/docker/index.md:120]: Looking for a bit more explanation here about the following command.

Is it: "The following example shows you how to build and run a Grafana image in Docker."? Or something similar?

Comment (DanCech) [reply docs/sources/setup-grafana/installation/docker/index.md:120]: We could say something like this:
```suggestion
The following example shows you how to build and run a custom Grafana docker image based on the latest official ubuntu-based Grafana docker image:
```

File: docs/sources/setup-grafana/installation/docker/index.md [hunk 2]
```diff
@@ -113,7 +113,20 @@ docker run -d \
 
 You can build your own customized image that includes plugins. This saves time if you are creating multiple images and you want them all to have the same plugins installed on build.
 
-In the [Grafana GitHub repository](https://github.com/grafana/grafana) there is a folder called `packaging/docker/custom/`, which includes two Dockerfiles, `Dockerfile` and `ubuntu.Dockerfile`, that can be used to build a custom Grafana image. It accepts `GRAFANA_VERSION`, `GF_INSTALL_PLUGINS`, and `GF_INSTALL_IMAGE_RENDERER_PLUGIN` as build arguments.
+In the [Grafana GitHub repository](https://github.com/grafana/grafana) there is a folder called `packaging/docker/custom/`, which includes a Dockerfile that can be used to build a custom Grafana image. It accepts `GRAFANA_VERSION`, `GF_INSTALL_PLUGINS`, and `GF_INSTALL_IMAGE_RENDERER_PLUGIN` as build arguments.
+
+The `GRAFANA_VERSION` build argument needs to be a valid `grafana/grafana` docker image tag. By default this will build an Alpine-based image. To build an Ubuntu-based image, append `-ubuntu` to the `GRAFANA_VERSION` build argument (Grafana v6.5+).
```
Comment (chri2547) [inline docs/sources/setup-grafana/installation/docker/index.md:118]:
```suggestion
The `GRAFANA_VERSION` build argument must be a valid `grafana/grafana` docker image tag. By default, this builds an Alpine-based image. To build an Ubuntu-based image, append `-ubuntu` to the `GRAFANA_VERSION` build argument (available in Grafana v6.5 and later).
```

**Review by chri2547 — COMMENTED** [review]
A suggestion and a question. Thank you for the contribution!

File: docs/sources/setup-grafana/installation/docker/index.md (hunk 1 above)
Comment (DanCech) [reply docs/sources/setup-grafana/installation/docker/index.md:120]: @chri2547 thoughts on ^?

Comment (chri2547) [reply docs/sources/setup-grafana/installation/docker/index.md:120]: Looks fine, just capitalize the two instances of Docker and capitalize Ubuntu.

**Review by chri2547 — APPROVED** [review]
lgtm! Thanks!

**Review by dsotirakis — APPROVED** [review]

**Comment by kminehart** [comment]
@DanCech if I run this with

```
docker build . --build-arg GRAFANA_TGZ=$PWD/grafana-enterprise-9.3.0.linux-amd64.tar.gz
```

I should expect this to build the docker image and skip all of the yarn isntall/yarn build/go gen/go build stuff and just use the assets from the tar file I provide, right? that doesn't seem to be happening :thinking:

**Comment by DanCech** [comment]
@kminehart this is in packaging/docker/custom/ which doesn't use GRAFANA_TGZ, it builds on top of an official image.

To use the standard Dockerfile to build from TGZ you have to use buildkit and set JS_SRC and GO_SRC build args like the build scripts do

**Comment by kminehart** [comment]
doh. for some reason I thought we also were able to get rid of the ones in `packaging` and only have 1 image for everything. Had a brain fart and thought that's what I was looking at.

**Pull request merged**